src/platform/posix/posix_resolv_gai.c

```diff
@@ -98,6 +98,7 @@ nni_posix_gai_errno(int rv)
 		return (nni_plat_errno(errno));
 
 	case EAI_NONAME:
+	case EAI_NODATA:
```

Comment:
This value, EA_NODATA, is not present everywhere, and is not part of the POSIX standard.  Therefore, it's use here needs to be guarded by an #ifdef EAI_NODATA macro check.

Reply:
Done as requested.